src/communication/msg.rs

```diff
@@ -58,6 +58,12 @@ impl From<String> for RequestId {
     }
 }
 
+impl From<&str> for RequestId {
+    fn from(id: &str) -> RequestId {
+        RequestId(IdRepr::String(id.to_string()))
+    }
+}
+
```

Comment:
Istnieje trait ze stringa, może warto z niego skorzystać, bo chcemy zrezygnować z kopiowania kodu z RA i dodać to jako dependency. Stąd niemożliwe będzie zaimplementowanie tego traita i łatwiej też będzie zrebasować potem

Reply:
Ok, poprawilem